Tools/autotest/sim_vehicle.py

```diff
@@ -577,6 +577,8 @@ def start_vehicle(binary, autotest, opts, stuff, loc):
             sys.exit(1)
         path += "," + str(opts.user_param_file)
         progress("Adding the user defined params from (%s)" % (str(opts.user_param_file),))
+    if path is None:
+        print("No default param file provide, abort launch.")
```

Comment:
Two points:

- no, don't abort launch, simply don't extend the cmd
- this is just printing, it didn't abort 😛

Reply:
oups I forget the exit(1).
If path == None, it  means that neither user param file nor default file is use. Therefore, no default parameters will be load by SITL. I assume that user using sim_vehicle.py aren't aware that SITL models need some parameters

Reply:
> that SITL models need some parameters

No, not necessarily. As I said, not all models have a defaults file now and run fine 😉 So no dying.

Reply:
I am surprise of that ! They need at least acc offset, unless you flight in sitl without arming check and with magic EKF_TYPE 10. 
But whatever, it revert that ! It shouldn't append anyway

Reply:
done!